src/stcal/jump/twopoint_difference.py

```diff
@@ -173,8 +173,6 @@ def find_crs(
         return gdq, row_below_gdq, row_above_gdq, 0, dummy
     else:
         # set 'saturated' or 'do not use' pixels to nan in data
-        dat[np.where(np.bitwise_and(gdq, sat_flag))] = np.nan
-        dat[np.where(np.bitwise_and(gdq, dnu_flag))] = np.nan
         dat[np.where(np.bitwise_and(gdq, dnu_flag + sat_flag))] = np.nan
```

Comment:
Is there any chance that either the dnu or sat flags might be set without both being set?

Reply:
The bottom line subsumes the previous two, right?  As long as dnu_flag and sat_flag are different, bitwise_and will be nonzero any time either of those bits is set, so np.where will pick out all points where either flag is set?  I tested this myself to be absolutely sure.

Reply:
In general, the syntax 
`np.where(np.bitwise_and(gdq, flag_a + flag_b + flag_c))`
should return all pixels where at least one of flags a, b, and c is set.  If it is clearer, we could write something like
`dat[np.bitwise_and(gdq, flag_a + flag_b + flag_c) != 0] = np.nan`
instead.

Reply:
Oh yes, you're right. I always struggle with bit-wise math.  Thanks!

Reply:
Sure!  The only hole this opens is if someone is foolish enough to make sat_flag and dnu_flag equal.  Hopefully that gets checked somewhere? :-)

Reply:
Pretty sure that's not possible, but to be super safe, we could cover that case by using `|` instead of `+`.